packages/contract/README.md

```diff
@@ -310,6 +310,52 @@ const val = instance.getValue();
 // since the contract returns that value.
 ```
 
+#### Overloaded functions
+
+You may find yourself in the situation of having a Truffle contract object that has multiple functions with the same name.
+You can call these "overloaded functions" just like you would a normal contract function. Truffle contract instances
+actually wrap web3's contract abstraction (`web3.eth.Contract`) and so when you call an overloaded function, it
+uses the same function resolution that web3 uses. However, we must give a warning that this overloaded function
+resolution is a bit dodgy and can resolve to the wrong function when you call it. So be careful!
+
+The more unambiguous way of calling these types of functions is to use the `methods` namespace on Truffle contract
```

Comment:
```suggestion
The precice way to invoke these types of functions is to use the `methods` namespace on Truffle contract
```

Reply:
Does it mean `precise` @cds-amal ?

Reply:
something is ambiguous when it is unclear - usually the meaning of something is ambiguous when it could mean one of several things

Reply:
for this one I re-wrote it as "The more precise way of referencing these...". Is that good?

Reply:
Yeah I think "unambiguous" here is clearer than precise.  I would just say "unambiguous" rather than "more unambiguous", though -- it *is* unambiguous, we don't need to qualify it.